mixer/test/e2e/e2e_cachebuster_test.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
```

Comment:
Actually, I am not sure which cache are we talking about here. Is it the expression evaluator cache ? Consider adding a comment about this test. It is unclear to me how are we trying to cause a cache miss.